testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite7NonGeneric.java

```diff
@@ -63,6 +64,10 @@ public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
                 testOnEnvironment(EnvSinglenodeSparkHive.class)
                         .withGroups("configured_features", "hive_spark")
                         .build(),
+                testOnEnvironment(EnvSinglenodeDeltaLakeOss.class)
+                        .withGroups("delta")
```

Comment:
why here? let's move it out to SuiteDeltaLake

Reply:
I think this is a left-over from the beginning of the PR when I was trying to see what is running on the CI.

Thank you for raising this.